### PR TITLE
update delimiting date content

### PR DESCRIPTION
**Pull request by egarreau**
screen shot 2018-03-30 at 11 45 38 am

### CODE REVIEW

File: content/pages/education/gi-bill/post-9-11.md [hunk 1]
```diff
@@ -55,7 +55,7 @@ If you qualify for more than one VA education benefit, you’ll have to pick whi
...
 
-**If your service ended after January 1, 2013:** There’s no time limit for using your education benefits (known as a “delimiting date”). But our automated system still prints letters with outdated information about end dates. Any delimiting date that’s on your records right now isn’t correct, but it’ll remain until we update our system. [Learn more about this and other new provisions from Forever GI Bill - Harry W. Colmery Veterans Educational Assistance Act.](https://www.benefits.va.gov/GIBILL/ForeverGIBill.asp)
+**If your service ended on or after January 1, 201:** There’s no time limit (known as a “delimiting date”) to use your education benefits. A recently enacted law gets rid of the expiration date for you. This change may not be reflected in some of the letters you receive from us. We're working to update our systems to support this change. We appreciate your patience as we make the updates to our system. [Learn more about this and other new provisions from Forever GI Bill - Harry W. Colmery Veterans Educational Assistance Act.](https://www.benefits.va.gov/GIBILL/ForeverGIBill.asp)
```
Comment (cehsu) [inline content/pages/education/gi-bill/post-9-11.md]: Is 201 correct here? Looping in @peggygannon, @bethpotts.

**Review by cehsu — COMMENTED** [review]
Looks good so far. I had a minor question about the date. We will probably want Peggy or Beth to sign off on this as well. Also, is the yarn.lock file part of these changes? It seems like it might not be related.

File: content/pages/education/gi-bill/post-9-11.md (hunk 1 above)
Comment (peggygannon) [reply content/pages/education/gi-bill/post-9-11.md]: @cehsu @egarreau I just responded to this on an issue from Evangeline. 201 should be 2013.

**Review by peggygannon — CHANGES_REQUESTED** [review]
@cehsu I responded to Evangeline in this issue: https://github.com/department-of-veterans-affairs/vets.gov-team/issues/9587#
--201 should be 2013
--please make the word system plural in this sentence: We appreciate your patience as we make the updates to our systems.

File: yarn.lock [hunk 2]
```diff
@@ -8242,6 +8242,10 @@ recursive-readdir@^2.1.0:
   dependencies:
     minimatch "3.0.3"
 
+recursive-uncache@^0.1.1:
```
Comment (cvalarida) [inline yarn.lock]: :thinking: Somebody else mentioned this was happening...Not sure why this is happening, but it doesn't look like it should be in this PR.

**Review by cvalarida — APPROVED** [review]
Looks good as updated!

**Review by peggygannon — APPROVED** [review]

**Pull request merged**